packages/smooth_app/lib/generic_lib/widgets/images/smooth_image.dart

```diff
@@ -28,17 +37,28 @@ class SmoothImage extends StatelessWidget {
   final BoxFit fit;
   final String? heroTag;
   final bool rounded;
+  final int? cacheWidth;
+  final int? cacheHeight;
```

Comment:
Those `cache` parameters may be helpful regarding memory, but not network latency.
Please consider implementing [my suggestion about `ImageSize.DISPLAY`](https://github.com/openfoodfacts/smooth-app/issues/5445#issuecomment-2193881775). That will dramatically help for memory too!

Reply:
Ok, I will.
I think, I will also add all variants of `ImageSize`

Reply:
Good. Btw I don't think all variants are always implemented; I would focus on 400 and FULL only.